src/App.tsx

```diff
@@ -1,54 +1,123 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
 import './App.scss';
+import classNames from 'classnames';
+import debounce from 'lodash.debounce';
 import { peopleFromServer } from './data/people';
```

Comment:
try to fix this linter error

![image](https://github.com/mate-academy/react_autocomplete/assets/91946036/865fdd7b-9c36-4d08-8ddb-baee98a89854)